src/simplenlg/lexicon/XMLLexicon.java

```diff
@@ -78,14 +79,14 @@
 	// added by vaudrypl
 	protected Map<LexicalCategory, List<WordElement>> indexByCategory; // map from variants
 
-	
+
 	/**********************************************************************/
 	// constructors
 	/**********************************************************************/
 
 	/**
 	 * Load an XML Lexicon from a named file
-	 * 
```

Comment:
After working at trying to not pollute the history with these whitespace changes, I gave up and so they are in there.